requirements.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/canonical/operator/#egg=ops
```

Comment:
Is there a reason we use the main branch of upstream directly instead of a pypi release?

Reply:
There used to be a feature that wasn't in the stable version of the package. I can't remember what it is now so I'll try to package the charm with the pypi ops package.

Reply:
It looks like it's been fixed now so I changed it to point to just `ops` and removed the build git part from charmcraft.yaml